doc/sphinx/source/howto/index.rst

```diff
@@ -11,7 +11,8 @@ How to
    operator.rst
    update.rst
    derive-for-training.rst
-   import.rst    
+   distribute-train.rst     
```

Comment:
typo in name? distributed vs distribute?

Reply:
@sfvaroglu ... we have this documentation under a section named `How to`.  So the action that falls after must make sense in a "How to" context.  I talked to @wenzhe-nrv about this, and we decided that "How to deploy distributed training" is accurate enough.  Do you agree?

Reply:
The file name I would like to keep abbreviated form, the title that Wenzhe wanted added was "How to train ..." whenever that link is referenced in text (shorter is better), it will spell out whatever the title is on that target page, which I think is better and helps us keep our docs consistent.